src/javaRosa.js

```diff
@@ -1393,6 +1393,9 @@ define([
             } else {
                 _this.__callOld();
             }
+            if (path.contains('casedb')) {
+                window.analytics.usage("Case Management", "Form Builder", "Click and Drag");
```

Comment:
Yep, for both.  Any time a case property is dragged out of the case property window.